pages/linux/sslstrip.md

```diff
@@ -0,0 +1,29 @@
+# sslstrip
+
+> Perform Moxie Marlinspike's Secure Sockets Layer (SSL) stripping attacks.
+> Perform an ARP spoofing attack in conjunction.
+> More information: <https://www.kali.org/tools/sslstrip/>.
+
+- Log only SSL POST traffic on port 10000 by default:
+
+`sslstrip`
+
+- Log only SSL POST traffic on specific port (e.g. 8080):
```

Comment:
`POST` is specific to HTTP/HTTPS. Maybe reword this?

Reply:
You're right, thanks a lot for the comment. Will do.